src/main/java/org/project36/qualopt/domain/Study.java

```diff
@@ -53,6 +53,17 @@
                joinColumns = @JoinColumn(name="studies_id", referencedColumnName="id"),
                inverseJoinColumns = @JoinColumn(name="participants_id", referencedColumnName="id"))
     private Set<Participant> participants = new HashSet<>();
+    
+    //Check this before sending email to new comer, only send emails to those that in this list
+    //Use @Transient because the database is not correctly configure for this yet, otherwise use @OneToMany
```

Comment:
Perhaps make a note that this is due to issue #45 as mentioned in your description here.